packages/scaffold/src/views/w3m-email-verify-otp-view/index.ts

```diff
@@ -16,12 +36,18 @@ export class W3mEmailVerifyOtpView extends W3mEmailOtpWidget {
         await this.emailConnector.provider.connectOtp({ otp })
         EventsController.sendEvent({ type: 'track', event: 'EMAIL_VERIFICATION_CODE_PASS' })
         await ConnectionController.connectExternal(this.emailConnector)
-        ModalController.close()
         EventsController.sendEvent({
           type: 'track',
           event: 'CONNECT_SUCCESS',
           properties: { method: 'email', name: this.emailConnector.name || 'Unknown' }
         })
+
+        const smartAccountEnabled = NetworkController.checkIfSmartAccountEnabled()
```

Comment:
If we don't have early returns in the function body, we could place all the variables on the top. This would help us to keep the functions in same convention like.

```js
if (this.emailConnector) {
        const smartAccountEnabled = NetworkController.checkIfSmartAccountEnabled()
        ...
        if (smartAccountEnabled && !this.smartAccountDeployed) { ... } 
```